EssentialFeed/EssentialFeed/Feed Presentation/FeedImageCommentsViewModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  FeedImageCommentsViewModel.swift
+//  EssentialFeed
+//
+//  Created by Ivan Ornes on 10/3/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public struct FeedImageCommentsViewModel {
+	public let comments: [FeedImageComment]
```

Comment:
The type here should be `[FeedImageCommentViewModel]`. The presenter should map from one type to the other.

```suggestion
	public let comments: [FeedImageCommentViewModel]
```